src/components/feed/FeedButton.js

```diff
@@ -0,0 +1,13 @@
+import "../../styles/FeedButton.css";
+
+const FeedButton = ({ onClick }) => {
+  return (
+    <>
+      <button className="FeedButton" onClick={onClick}>
+        질문 작성하기
+      </button>
+    </>
```

Comment:
불필요한 프래그먼트도 다음에 다같이 정리하죠 ㅎㅎ
```suggestion
    return (
      <button className="FeedButton" onClick={onClick}>
        질문 작성하기
      </button>
    )
```